src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/index.js

```diff
@@ -36,12 +37,16 @@ const typesToConvert = {
   social_embed: socialEmbed,
 };
 
-const parseBlockByType = (block, json) => {
+const parseBlockByType = (block, json, assetType) => {
   if (!path(['type'], block)) return false;
 
   const { type } = block;
 
-  const parsedBlock = (typesToConvert[type] || handleMissingType)(block, json);
+  const parsedBlock = (typesToConvert[type] || handleMissingType)(
+    block,
+    json,
+    assetType,
+  );
```

Comment:
I know we're just extending what's already there but I wondered if this would be more readable if we moved `handleMissingType` above L47. What do you think?